pkg/cloud/services/securitygroup/securitygroups_test.go

```diff
@@ -1078,6 +1079,122 @@ func TestAdditionalControlPlaneSecurityGroup(t *testing.T) {
 	}
 }
 
+func TestAdditionalManagedControlPlaneSecurityGroup(t *testing.T) {
+	scheme := runtime.NewScheme()
+	_ = ekscontrolplanev1.AddToScheme(scheme)
+
+	testCases := []struct {
+		name                         string
+		networkSpec                  infrav1.NetworkSpec
+		expectedAdditionalIngresRule infrav1.IngressRule
+	}{
+		{
+			name: "default control plane security group is used",
+			networkSpec: infrav1.NetworkSpec{
+				AdditionalControlPlaneIngressRules: []infrav1.IngressRule{
+					{
+						Description: "test",
+						Protocol:    infrav1.SecurityGroupProtocolTCP,
+						FromPort:    9345,
+						ToPort:      9345,
+					},
+				},
+			},
+			expectedAdditionalIngresRule: infrav1.IngressRule{
```

Comment:
typo? Ingress

Reply:
Indeed, I copied from the existing test but didn't notice. I've fixed it, thanks.